components/searchable_user_list/searchable_user_list.tsx

```diff
@@ -169,8 +170,8 @@ class SearchableUserList extends React.PureComponent {
     render() {
         let nextButton;
         let previousButton;
-        let usersToDisplay;
-        const {formatMessage} = this.props.intl;
+        let usersToDisplay = this.props.users;
+        const formatMessage = this.props.intl;
```

Comment:
This is causing errors on the test server since intl is still an object containing a `formatMessage` function